persistence/src/commonMain/db_user/com/wire/kalium/persistence/Conversations.sq

```diff
@@ -85,6 +85,13 @@ UPDATE Conversation
 SET muted_status = ?, muted_time = ?
 WHERE qualified_id = ?;
 
-
 updateAccess:
 UPDATE Conversation SET access_list= ?, access_role_list = ? WHERE qualified_id = ?;
+
+deleteAllMessages:
+DELETE FROM Message
+WHERE conversation_id IS :conversation_id;
+
+getMessagesByContentType:
```

Comment:
I think the name could be something like: `getAssetsMessagesByConversationId`. Only if we are not expected to pass a different content type in the query filter